lib/charms/mysql/v0/mysql.py

```diff
@@ -498,9 +498,15 @@ def create_cluster(self, unit_label: str) -> None:
 
         Raises MySQLCreateClusterError if there was an issue creating the cluster.
         """
+        # defaulting group replication communication stack to MySQL instead of XCOM
```

Comment:
should we be importing the mysql lib from the vm charm here?

Reply:
I've kind of done that, not from the published/main branch, but from the TLS enabled branch. Since that is currently blocked and we need this change, I've just copied the file from the branch, we eventually will overwrite from VM once all TLS support get merged over there.